Validation/HGCalValidation/test/python/testValidationHGCalTrigPrim_RelVal_cfg.py

```diff
@@ -16,9 +16,8 @@
 process.load('FWCore.MessageService.MessageLogger_cfi')
 process.load('Configuration.EventContent.EventContent_cff')
 process.load('SimGeneral.MixingModule.mixNoPU_cfi')
-#process.load('Configuration.Geometry.GeometryExtended2023D17Reco_cff')
-#process.load('Configuration.Geometry.GeometryExtended2023D17_cff')
 process.load('Configuration.Geometry.GeometryExtended2026D49Reco_cff')
+process.load('Configuration.Geometry.GeometryExtended2026D49_cff')
```

Comment:
D49 is obsolete since long, and these configurations are not in the release anymore.